packages/jobs/lib/execution/action.ts

```diff
@@ -135,7 +135,9 @@ export async function startAction(task: TaskAction): Promise<Result<void>> {
             runnerFlags: await getRunnerFlags(),
             startedAt: now,
             endUser,
-            heartbeatTimeoutSecs: task.heartbeatTimeoutSecs
+            heartbeatTimeoutSecs: task.heartbeatTimeoutSecs,
+            oauth_client_id: providerConfig.oauth_client_id,
```

Comment:
I would prefer to have a field of type Partial< Config > or at least a matching object that we can extends rather than free-floating fields